src/main/webapp/index-script.js

```diff
@@ -390,10 +392,10 @@ function populateReviews(reviewList, placeID) {
   const entireList = document.createElement('ul');
   entireList.id += 'reviews-list';
 
-  if (reviewList == null) {
+  if (reviewList[0] == null) {
```

Comment:
Can this array be empty? If so, you might want to check the length before indexing into array.

Reply:
Sounds good, I can fix that in my next PR since i'll be building on this anyways